frontend/src/app/modules/work-space/work-space-routing.module.ts

```diff
@@ -49,8 +49,7 @@ const routes = [
       {
         path: '**',
         component: NotFoundComponent,
-        pathMatch: 'full',
-        skipLocationChange: true,
+        pathMatch: 'full'
       },
```

Comment:
Why did you remove "skipLocationChange" ? It was set to get original url on the NotFound page